docs/content/latest/benchmark/performance-troubleshooting.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Troubleshooting performance issues in YugabyteDB
+headerTitle: Performance Troubleshooting
+linkTitle: Performance Troubleshooting
+description: Common steps to take when investigating performance issues in YugabyteDB
+image: /images/section_icons/architecture/concepts.png
+headcontent: Performance Troubleshooting
+menu:
+  latest:
+    identifier: large-datasets-2-ysql
+    parent: benchmark
+    weight: 7
+showAsideToc: true
+isTocNested: true
+---
+
+Use this page for general guidance and steps you can take when troubleshooting the performance of your Yugabyte cluster.
+
+## Location of various files on a YB cluster
```

Comment:
I would probably also note that these default locations, relative to `/home/yugabyte`, are true for platform, otherwise, it's up to the user where they install YB, vs where they point their data dirs to